java-frontend/src/main/java/org/sonar/java/se/constraint/ConstraintManager.java

```diff
@@ -119,25 +119,25 @@ public SymbolicValue createBinarySymbolicValue(Tree syntaxNode, List<SymbolicVal
   }
 
   private SymbolicValue not(RelationalSymbolicValue relationalSymbolicValue) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make "not" a "static" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325)